### PR TITLE
Permission updates: HbxAdminController.

### PR DESCRIPTION
**Pull request by TreyE**
https://www.pivotaltracker.com/story/show/187177642

### CODE REVIEW

File: app/policies/admin/aptc_policy.rb
Comment (nisanthyaganti9) [inline app/policies/admin/aptc_policy.rb:7]: this should be `staff_can_edit_aptc?`; we will have these methods available in application policy, and we're explicitly defining one for each permission.

File: app/policies/application_policy.rb
Comment (nisanthyaganti9) [inline app/policies/application_policy.rb:294]: this can be removed; we are explicitly defining these.

**Review by nisanthyaganti9 — COMMENTED** [review]
This needs to wait until we merge admin related changes that i'm on working on.

File: app/controllers/hbx_admin_controller.rb
Comment (saikumar9) [inline app/controllers/hbx_admin_controller.rb:11]: @TreyE 
* We are following a pattern of matching the route with the policy method by adding `?` at the end. Can you make the code change to match the same pattern?
* Curious to know why you added two policies instead of one `HbxAdminPolicy`?

**Review by saikumar9 — CHANGES_REQUESTED** [review]

File: app/controllers/hbx_admin_controller.rb
Comment (TreyE) [reply app/controllers/hbx_admin_controller.rb:11]: I can easily make the change.  Typically in ruby the predicate methods end in ? - I was trying to follow that convention.

The reason I made two different policies is because we're accessing two different types of resources - one is testing if someone is allowed to perform administrative APTC actions - and the other is for access to EnrollRegistry settings.  They should, at the end of the day, probably be in different controllers, as they are different resources.  But for now I wanted to set up proper access control on the ability to see the settings - event if eventually it lives somewhere else.

Comment (saikumar9) [reply app/controllers/hbx_admin_controller.rb:11]: Maybe I was not clear on the first point. I am suggesting using the `registry?` method instead of `show?` for the endpoint `registry` for pundit authorization.

**Review by saikumar9 — APPROVED** [review]
lgtm!

**Review by harshaellanki — APPROVED** [review]

**Pull request merged**
